test_config.yml

```diff
@@ -0,0 +1,39 @@
+# yaml file to configure the Landsat C2 SR & ST parameters for this repository workflow
+
+# The following parameters are required to be set by the user unless otherwise indicated.
+
+local_settings:
+- data_dir: "~/Documents/GitHub/Landsat_C2_SRST/data_acquisition/in/" # point to the directory where your location information is stored - this path must end with a '/'
```

Comment:
Should/can this path be relative to the project directory? When I ran `tar_make()` it tried unsuccessfully to find this file because this was not where the project dir was located.

Reply:
Ah...I see now the comment in `config.yml` lol

Reply:
In my original thought, folks could point to files outside of the repo (thinking about shapefiles and the like), but maybe it would be easier to just say that part of the preparation to run the workflow is to put input files at `data_acquisition/in`?

Reply:
I'm going to leave this as-is for the time being, but again, philosophical conversation needed. :)